packages/ember-data/lib/system/model/states.js

```diff
@@ -417,6 +417,12 @@ createdState.uncommitted.rollback = function(record) {
   record.transitionTo('deleted.saved');
 };
 
+createdState.uncommitted.pushedData = function(record) {
+  record.transitionTo('loaded.saved');
```

Comment:
`loaded.saved` may be the wrong state here. If there are unsaved changes in `_attributes` we likely want to transition to `loaded.updated.uncommitted`